src/ui-kit/SaveButton/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { Text, TouchableOpacity } from 'react-native';
+
+import buttonsize from '../SaveButton/size';
+
+const SaveButton = ({ onPass, title, small }) => {
```

Comment:
Change the component and file name. Now it's already a global UI button, not a "save" button